template/toc.html.js

```diff
@@ -23,7 +23,14 @@ exports.transform = function (model) {
     item.topicHref = item.topicHref || null;
     item.tocHref = item.tocHref || null;
     item.name = item.name || null;
-
+    if(item.new || item.updated) {
+      item.new = true;
+      item.newType = item.updated ? 'updated' : 'new';
+      item.newTypeLabel = item.newType ? item.newType.toUpperCase() : null;
```

Comment:
Let's use `labelType` instead of `newType` and `labelText` instead of `newTypeLabel`. Also consider using of enumerator for the item labels. For example:

```
const labels = {
    NEW: 'NEW',
    UPDATED: 'UPDATED',
    DEPRECATED: 'DEPRECATED label text example' // for the purposes of this example, if one day we decide to add a new label type 
}
```

..

```
// JS Code after the change
item.labelType = item.updated ? labels.UPDATED : labels.NEW;
item.newTypeLabel = item.newType ? item.labelType : null;
```

Reply:
Done